src/sxastarter/src/components/play-travel/Accordion.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Box,
+} from '@chakra-ui/react';
+
+import {
+  ComponentParams,
+  ComponentRendering,
+  Text as JssTextField,
+  Image as JssImageField,
+  TextField,
+  RichTextField,
+  ImageField,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+
+interface AccordionItemProps {
+  heading: { jsonValue: TextField };
+  content: { jsonValue: RichTextField };
+  image: { jsonValue: ImageField };
+}
+
+interface Fields {
+  data: {
+    accordion: {
+      children: {
+        results: AccordionItemProps[];
+      };
+    };
+  };
+}
+
+interface AccordionProps {
+  rendering: ComponentRendering & { params: ComponentParams };
+  params: ComponentParams;
+  fields: Fields;
+}
+
+const AccordionItemRendering = (props: AccordionItemProps) => (
+  <AccordionItem>
+    <h2>
+      <AccordionButton
+        className={'h-20'}
+        _hover={{
+          bgColor: 'white',
+        }}
+      >
+        <Box className={'flex justify-between font-bold w-full'} fontSize={'28px'}>
+          <JssTextField field={props.heading.jsonValue} />
+          <AccordionIcon color={'#515151'} />
+        </Box>
+      </AccordionButton>
+    </h2>
+    <AccordionPanel
+      borderTopWidth={`1px`}
+      paddingBottom={{ base: '20px', md: '28px' }}
```

Comment:
lets use REMs instead of PXs because everywhere we use exactly this size system

So, 20px is 1.25rem
28px is 1.5rem

please check it everywhere